longhaul-test/feed-generator-deploy.yml

```diff
@@ -3,6 +3,22 @@
 # Licensed under the MIT License.
 # ------------------------------------------------------------
 
+kind: Service
```

Comment:
You do not need service because you set prom annotation in replicaset..

Reply:
Prom can't connect to the app without this, but if there's a better way I would be interested.